index.html

```diff
@@ -2,11 +2,158 @@
 <html lang="ru">
   <head>
     <meta charset="utf-8">
-    <title>HTML Academy: Глейси</title>
+    <title>Магазин мороженого "GllacyShop"</title>
+    <link rel="icon" href="favicon.ico"> <!-- 32/32 -->
+    <link rel="icon" href="img/favicon/152.svg" type="image/svg+xml"> <!-- 152/152 -->
+    <link rel="apple-touch-icon" href="img/favicons/apple.png">  <!-- 180×180 -->
+    <link rel="manifest" href="manifest.webmanifest">
   </head>
   <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
-
+   <header>
+    <!--<p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p> -->
+     <nav>
+       <a><img src="img/Logo.svg" width="154" height="64" alt=""></a> <!--логотип магазина-->
+        <ul>
+           <li><a href="catalog.html">Каталог</a></li>
+           <li><a href="blank.html">Доставка и оплата</a></li>
+           <li><a href="blank.html">О компании</a></li>
+         </ul>
+      <ul>
+           <li><a href="blank.html">Поиск</a></li>
+           <li><a href="blank.html">Вход</a></li>
+           <li><a href="blank.html">Корзина</a></li>
+      </ul>
+      <p>с 10 до 20, ежедневно</p>
+      <p><a href="tel:+78124502525">8 812 450-25-25</a></p>
+  </nav>
+ </header>
+    <main>
+     <h1>Магазин мороженого Gllecy Shop</h1>
+     <section> <!-- Основная картинка -->
+       <img src="img/Group1.png" width="647" height="584" alt="Два рожка мороженого">
```

Comment:
```suggestion
       <img src="img/group1.png" width="647" height="584" alt="Два рожка мороженого">
```
Не всегда в макете названия файлов имеют содержательные имена. Поэтому, если имя файла неинформативное, как в данном случае **Group1.png** (что за группа1 ?), то их лучше переименовать по своему.